pages/understanding-json-schema/reference/type.md

```diff
@@ -175,3 +175,129 @@ types. For example, numeric types have a way of specifying a numeric
 range, that would not be applicable to other types. In this reference,
 these validation keywords are described along with each of their
 corresponding types in the following chapters.
+
+## Format[#format]
+
+The `format` keyword allows for basic semantic identification of certain
+kinds of values that are commonly used. or user could define its own format and use them alongside
```

Comment:
Nitpicking: A typo + not in present tense
Suggestion: kinds of commonly used, or a user could define their format and use them alongside